src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -21,20 +21,29 @@
  */
 class Crawler implements \Countable, \IteratorAggregate
 {
+    /**
+     * @var string|null
+     */
     protected $uri;
 
     /**
-     * @var string The default namespace prefix to be used with XPath and CSS expressions
+     * The default namespace prefix to be used with XPath and CSS expressions.
+     *
+     * @var string
```

Comment:
Moving the description above the `@var` annotation is the recommended way of describing a property. Additionally, that will keep the diff simpler if we remove the `@var` annotation when migrating to typed properties.